drivers/vulkan/rendering_device_driver_vulkan.cpp

```diff
@@ -48,6 +48,8 @@
 
 #define ARRAY_SIZE(a) std_size(a)
 
+#define VULKAN_RAYTRACING_ENABLED() !(defined(MACOS_ENABLED) || defined(IOS_ENABLED))
```

Comment:
Minor nitpick, I don't think we usually add () on macros that don't take arguments.